eng/Versions.props

```diff
@@ -14,6 +14,8 @@
     <MicrosoftAspNetCoreApp21Version>$(MicrosoftNETCoreApp21Version)</MicrosoftAspNetCoreApp21Version>
     <MicrosoftNETCoreApp31Version>3.1.8</MicrosoftNETCoreApp31Version>
     <MicrosoftAspNetCoreApp31Version>$(MicrosoftNETCoreApp31Version)</MicrosoftAspNetCoreApp31Version>
+    <MicrosoftNETCoreApp50Version></MicrosoftNETCoreApp50Version>
+    <MicrosoftAspNetCoreApp50Version></MicrosoftAspNetCoreApp50Version>
     <!-- Latest shared runtime version updated by darc -->
     <MicrosoftNETCoreAppVersion>5.0.0-rtm.20529.3</MicrosoftNETCoreAppVersion>
```

Comment:
We should probably start moving to 6.0 by now. Is there much churn at all in 5.0 anymore?

Reply:
Don't we need that stable installed version "5.0.0" to match the package version?  I was wanting for that to happen before changing these properties.  The new 5.0 properties are not set up to deal with the install/package version difference.

Reply:
Ah, if we don't support the difference then all good. For some other time. There's plenty to do